### PR TITLE
Add pfsarch and storage to config at IPMU.

### PR DESCRIPTION
**Pull request by yukimoritani**
following the fact that pfsarch will be operated under PFS-LAN at IPMU.

### CODE REVIEW

**Comment by himorin** [comment]
could you fine a relevant ticket to JIRA on this PR?

File: dnsmasq-site.ipmu [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #  02 : (not assigned) CURRENTLY RESERVED
 #  03 : External service
 #  04 : VMs for normal works
-#  05 : (not assigned)
+#  05 : Dara aichive (pfsarch)
```
Comment (himorin) [inline dnsmasq-site.ipmu]: Dara aichive -> Data archive

**Comment by yukimoritani** [comment]
Well ... when we talked this matter two days ago, I hearrd I don't need to file a ticket for this change.
But maybe I misunderstood you.
I'm sorry about that.

**Comment by yukimoritani** [comment]
I agree.
I just filed the ticket now, but INFRA... I hope it is OK..

File: macs-ipmu/srv.conf [hunk 2]
```diff
@@ -23,9 +23,12 @@
...
 
 # Data Archive
-d4:ae:52:d0:67:b9,pas-srv
 00:D0:23:85:4C:05,pad1-srv
 00:D0:23:45:4C:05,pad2-srv
 00:D0:23:C5:4C:05,pad3-srv
 00:D0:23:25:4C:05,pad4-srv
 
+# pfsarch / raa2-vm (r210-2)
+d4:ae:52:d0:67:b8,raa2-vm
```
Comment (himorin) [inline macs-ipmu/srv.conf]: could you switch from -vm to -srv, to match with the main interface?

**Pull request merged**